spec/features/user_browses_by_category_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+feature 'any user is able to see' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.